Differentiated between local folder and github.com MG

Pandoraaaaa opened the pull request:
I have not gone systematically through the whole file, but I made changes to the areas discussed at our last meeting, regarding the location of a repo. MG

John-Doeeeee commented:
I think "html-examples" should be changed to "Workflow" throughout.
Add Section on "Inviting a collaborator"
Add section on branching.
Add section on cloning.

Pandoraaaaa commented:
I have received your to-do list. MG

John-Doeeeee commented:
Does this work? J*

On Fri, Oct 6, 2017 at 6:33 AM, Pandoraaaaa <notifications@github.com>
wrote:

> I have received your to-do list. MG
>
> —
> You are receiving this because your review was requested.
> Reply to this email directly, view it on GitHub
> <https://github.com/John-Doeeeee/Workflow/pull/1#issuecomment-334756142>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/Admk4V1dK26qm76x26wcWpvHQM43xeEZks5spiwngaJpZM4PwYFg>
> .
>

Pandoraaaaa commented:
Comments cause emails to be sent, but the initial pull request does not cause an email. Rather you must request that your collaborator review the pull request to cause an email to be sent.

Pandoraaaaa commented:
I've modified the workflow.txt document to use "Workflow" rather than "html-examples". Also, I've added a new document called Addendum that describes cloning and branching.

I used MS Wordpad to make the changes and saved in a format called Office Open XML. 

MG
